wormhole-connect/src/utils/routes/bridge.ts

```diff
@@ -322,4 +322,21 @@ export class BridgeRoute extends BaseRoute {
       Buffer.from(messageInfo.rawVaa).toString(),
     );
   }
+
+  async nativeTokenAmount(
+    destChain: ChainName | ChainId,
+    token: TokenId,
+    amount: BigNumber,
+    walletAddress: string,
+  ): Promise<BigNumber> {
+    throw new Error('Not implemented');
```

Comment:
'Not supported'

Reply:
Can we add a const at the top of the class that says whether native gas dropoff is a supported feature of that route?

Reply:
yup done!